File: MORM.Php/Classes/parametro.php
<?php

define("DATABASE", "igreja");
define("USERNAME", "root");
define("PASSWORD", "");
define("HOSTNAME", "localhost");

class Parametro
{
    private static $instance;

    public static function Instance()
    {
        if (!isset(self::$instance))
            self::$instance = new Parametro(DATABASE, USERNAME, PASSWORD, HOSTNAME);
        return self::$instance;
    }
    
    //--
    
    public function __construct($database, $username, $password, $hostname)
    {
        $this->Database = $database;
        $this->Username = $username;
        $this->Password = $password;
        $this->Hostname = $hostname;
    }
    
    public $Hostname;
    public $Username;
    public $Password;
    public $Database;    
}
?>

File: MORM.CSharp/MORM.CSharp/Classes/Parametro.cs
using System.Configuration;

namespace MORM.CSharp.Classes
{
    public class Parametro
    {
    	public Parametro(string database, string username, string password, string hostname)
        {
            Database = database;
            Username = username;
            Password = password;
            Hostname = hostname;
        }
        
    	public Parametro()
        {
    		Database = ConfigurationManager.AppSettings["database"];
            Username = ConfigurationManager.AppSettings["username"];
            Password = ConfigurationManager.AppSettings["password"];
            Hostname = ConfigurationManager.AppSettings["hostname"];
        }

    	public string Database { get; private set; }
        public string Username { get; private set; }
        public string Password { get; private set; }
        public string Hostname { get; private set; }
    }
}